modules/styling-transform/package.json

```diff
@@ -41,7 +41,7 @@
   ],
   "dependencies": {
     "@emotion/serialize": "^1.0.2",
-    "@workday/canvas-kit-styling": "^10.2.5",
+    "@workday/canvas-kit-styling": "^10.3.0",
```

Comment:
that package depends on itself?

Reply:
this is the styling-transform package. They're different.